docs/references/react-router/overview.mdx

```diff
@@ -4,7 +4,7 @@ description: Learn how to integrate Clerk into your React Router application usi
 ---
 
 > [!WARNING]
-> The React Router SDK is currently in beta.
+> Due to an upstream issue with [React Router](https://github.com/remix-run/react-router/issues/12475), you'll need to use Node.js 22.11 or lower until the issue is resolved.
```

Comment:
Not sure where's the best place to put another callout so I had to remove the current one, multiple warnings stacked looks weird 🤔  Feel free to update!